esp32c3/src/assist_debug.rs

```diff
@@ -40,7 +40,7 @@ pub struct RegisterBlock {
     log_mem_writing_addr: LOG_MEM_WRITING_ADDR,
     log_mem_full_flag: LOG_MEM_FULL_FLAG,
     c0re_0_lastpc_before_exception: C0RE_0_LASTPC_BEFORE_EXCEPTION,
-    c0re_0_debug_mode: C0RE_0_DEBUG_MODE,
+    core_0_debug_mode: CORE_0_DEBUG_MODE,
```

Comment:
Please also fix the `lastpc` stuff that you can see one line up.

Reply:
Missed that, updated, thanks!